pod/perldata.pod

```diff
@@ -137,11 +137,12 @@ That is, any word character in the ASCII range, as long as the first
 character is not a digit.
 
 There are two package separators in Perl: A double colon (C<::>) and a single
-quote (C<'>).  Normal identifiers can start or end with a double colon, and
-can contain several parts delimited by double colons.
-Single quotes have similar rules, but with the exception that they are not
-legal at the end of an identifier: That is, C<$'foo> and C<$foo'bar> are
-legal, but C<$foo'bar'> is not.
+quote (C<'>).  Use of C<'> as the package separator is deprecated and will be
+removed in Perl 5.40.  Normal identifiers can start or end with a double
```

Comment:
Maybe it would be better to say "Normal identifiers can start with a double colon, (in which case the leading double colon is short hand for 'main::'), and trailing colons reference the glob or slots of the glob which represents the package of the name without the trailing colons."

Or perhaps something a little less clumsy that gets to the same point.

IOW, I would not consider a variable with a trailing colon to be a "normal identifier". It isnt. `$foo::bar::` is another way to access `${*foo::bar::{SCALAR}}`, and doesn't have a normal identifier name. Afaik the only way to address this variable is via its trailing double colons. Compare with `$foo::bar::baz`, which represents the SCALAR slot of the `baz` glob in `%foo::bar::`, which itself represents the hash slot of the glob `*foo::bar::`.

```
perl -le'$foo::bar::baz = "baz"; $foo::bar::= "BLORP"; print for keys %foo::bar::; print "---"; print for keys %foo::; print "---"; print ${*foo::bar::{SCALAR}}'
baz
---
bar::
---
BLORP
```

Reply:
I made no changes to that wording, I only word-wrapped the resulting paragraph after adding the deprecation text.